src/main/java/school/lemon/changerequest/java/Accountant.java

```diff
@@ -0,0 +1,28 @@
+package school.lemon.changerequest.java;
+
+
+/**
+ * Created by lera on 15.12.16.
+ */
+public class Accountant extends Employee {
+
+    private Employee[] employees;
+
+    public Accountant(Employee[] employees) {
+        super();
+        this.employees = employees;
+    }
+
+    public Accountant(String name) {
+        super(name);
+    }
+
+    public double calcSalary() {            //calculate overall salary for all accountable employees (including himself)
```

Comment:
Your accountant do not count himself salary. He only counts salary of employees from `employees` array, but as in your Demo class, you do not add accountant on which `calcSalary` method was called to the `employees` array.